pom.xml

```diff
@@ -484,5 +502,80 @@
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>unpack-jboss-as</id>
+                                <goals>
+                                    <goal>unpack</goal>
+                                </goals>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <!-- Disable downloaded WildFly distribution filesystem setup -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>rename-unpacked-jboss-as</id>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <phase>none</phase>
+                            </execution>
+                            <execution>
+                                <id>initialize-basedirs</id>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+            <dependencyManagement>
+                <dependencies>
+                    <!-- Bootable JAR related dependencies -->
+                    <dependency>
+                        <groupId>org.wildfly.arquillian</groupId>
+                        <artifactId>wildfly-arquillian-container-bootable</artifactId>
+                        <version>${version.org.wildfly.arquillian}</version>
+                        <exclusions>
+                            <exclusion>
+                                <groupId>org.jboss.sasl</groupId>
+                                <artifactId>jboss-sasl</artifactId>
+                            </exclusion>
+                            <!-- TODO remove this exclusion once this depends on Jakarta Inject -->
+                            <!-- superseded by jakarta.inject:jakarta.inject-api -->
+                            <exclusion>
+                                <groupId>javax.inject</groupId>
+                                <artifactId>javax.inject</artifactId>
+                            </exclusion>
+                        </exclusions>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.wildfly.core</groupId>
+                        <artifactId>wildfly-launcher</artifactId>
```

Comment:
Btw, what is this dependency about? It is used in 4 modules out of 7. Can it be used for all 7 (although not needed for all of them) and be defined directly in main pom.xml (to decrease maintenance for 4 same definitions in 4 different places)?

Reply:
It is the API for WIldFly launch. For instance a new command builder to start WildFly has been introduced in order to support the execution in case of bootable JAR. This is needed by the WildFly Arquillian bootable container adapter.

Reply:
Seems good to me as it is currently - using just a reference from dependency management.